source/common/http/async_client_impl.cc

```diff
@@ -126,6 +126,11 @@ AsyncStreamImpl::AsyncStreamImpl(AsyncClientImpl& parent, AsyncClient::StreamCal
       retry_policy_(createRetryPolicy(parent, options, parent_.factory_context_, creation_status)),
       account_(options.account_), buffer_limit_(options.buffer_limit_), send_xff_(options.send_xff),
       send_internal_(options.send_internal) {
+  // A field initialization may set the creation-status as unsuccessful.
```

Comment:
s/field/failed?

Reply:
Field, or data-member.

Reply:
I see, the field is `retry_policy_`, would be better to explicitly mention it, but not a big deal. Thanks!

Reply:
I'm assuming that in the future we will have more fields that may fail (as more exceptions are removed), so I've decided to keep the comment more generic.